src/clj/y_video_back/routes/service_handlers/handlers/media_handlers.clj

```diff
@@ -6,10 +6,15 @@
    [y-video-back.db.users :as users]
    [y-video-back.routes.service-handlers.utils.utils :as utils]
    [y-video-back.routes.service-handlers.utils.role-utils :as ru]
-   [ring.util.response :refer [file-response]]
-   [y-video-back.log :as log-ut]))
+   [ring.util.response :as rs :refer [file-response]]
+   [ring.mock.request :as mr]
+   [y-video-back.log :as log-ut]
+   [taoensso.timbre :as log]))
 
-; TODO - check if user has permission to stream requested file
+                                        ; TODO - check if user has permission to stream requested file
+
+(defn extension [s]
```

Comment:
I don't know if using short variables like `s` (for "string"?) is conventional in Clojure, but I think more verbose variable names are almost always readable/maintainable. The only exception I make for myself is when it is strongly suggests by context, e.g. `for s in strings`.